jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -1008,6 +1014,22 @@ def triggerBWCTests(String buildManifestUrl, String platform, String distributio
     }
 }
 
+def triggerDistributionValidationWorkflow(String version) {
```

Comment:
1. Can we add a param for people to disable this behavior.
2. if OSD built before OS and failed validation due to OS being old version or not exist, how do we avoid such issue?

Reply:
+1. We should have a mechanism to disable the validation.
Besides that, I'm thinking do we skip this validation if the workflow itself fails?

Reply:
1. We want the RC build to fail if validation fails for any reason. We added too many params at this point and RM usually forgets what to check mark and what not to. Example: docker build-with-tag. (Forgot it many times)
2. I discussed with @rishabh6788 in order to not wait for entire OpenSearch build to complete, I passed the `latest` to validation which should still be okay. Else we would have to wait for backend to complete building.